lua/lspconfig/server_configurations/julials.lua

```diff
@@ -1,5 +1,34 @@
 local util = require 'lspconfig.util'
 
+local function activate_env()
+  local bufnr = vim.api.nvim_get_current_buf()
+  local julials_client = util.get_active_client_by_name(bufnr, 'julials')
+  if julials_client then
+    local julia_project_files = { 'Project.toml', 'JuliaProject.toml' }
+    local depot_envs = vim.fs.normalize(vim.env.JULIA_DEPOT_PATH or vim.fn.expand '~/.julia') .. '/environments'
+    local environments = {}
+    vim.list_extend(environments, vim.fs.find(julia_project_files, { type = 'file', upward = true, limit = math.huge }))
+    vim.list_extend(
+      environments,
+      vim.fs.find(function(name, path)
+        return vim.tbl_contains(julia_project_files, name) and string.sub(path, #depot_envs + 1):match '^/[^/]*$'
+      end, { path = depot_envs, type = 'file', limit = math.huge })
+    )
+    environments = vim.tbl_map(util.path.dirname, environments)
+    vim.ui.select(environments, { prompt = 'Select a Julia environment' }, function(environment)
+      if environment then
+        julials_client.notify('julia/activateenvironment', { envPath = environment })
+        vim.notify('Julia environment activated: \n`' .. environment .. '`', vim.log.levels.INFO)
+      end
+    end)
+  else
+    vim.notify(
+      'method julia/activateenvironment is not supported by any servers active on the current buffer',
+      vim.log.levels.WARN
+    )
+  end
+end
```

Comment:
You can first check whether
```lua
if not julia_client then
      return vim.notify
end
```

Reply:
Revised the commit to follow this pattern. Thanks for the suggestion